src/Ledger/Transaction.lagda

```diff
@@ -235,8 +235,12 @@ Ingredients of the transaction body introduced in the Conway era are the followi
 \end{NoConway}
 
 \begin{code}[hide]
-  isP2Script : Script → Bool
-  isP2Script = is-just ∘ isInj₂
+  isP2Script : Script → Type
+  isP2Script = From-inj₂
```

Comment:
`From-inj₂` is incorrect here, since it's always inhabited whereas the original code gives a `false` on the `inj₁` constructor. It's a shame the stdlib doesn't have something like `IsInj₂`, defined like `From-inj₂` but with a `⊥` instead of a `⊤`.